WorkloadManagementSystem/Utilities/QueueUtilities.py

```diff
@@ -0,0 +1,225 @@
+"""Utilities to help Computing Element Queues manipulation
+"""
+
+__RCSID__ = '$Id$'
+
+from DIRAC import S_OK, S_ERROR
+from DIRAC.Core.Utilities.List import fromChar
+from DIRAC.Core.Utilities.ClassAd.ClassAdLight import ClassAd
+from DIRAC.ConfigurationSystem.Client.Helpers.Resources import getDIRACPlatform
+from DIRAC.ConfigurationSystem.Client.Helpers.Operations import Operations
+from DIRAC.Core.Security.ProxyInfo import getProxyInfo
+
+
+def getQueuesResolved(siteDict):
+  """
+  Get the list of queue descriptions merging site/ce/queue parameters and adding some
+  derived parameters.
+
+  :param dict siteDict: dictionary with configuration data as returned by Resources.getQueues() method
+
+  :return: S_OK/S_ERROR, Value dictionary per queue with configuration data updated, e.g. for SiteDirector
+  """
+
+  queueDict = {}
+
+  for site in siteDict:
+    for ce in siteDict[site]:
```

Comment:
```suggestion
    for ce, ceDict in siteDict[site].items():
```
and drop the next line

Reply:
OK